<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {	
	background-color: qlineargradient(
    	spread:pad, 
    	x1:0, y1:0, x2:1, y2:0,
   		stop:0 rgb(0, 49, 71),
    	stop:0.4 rgb(0, 49, 71),
    	stop:0.401 rgb(3, 169, 244),
   		stop:1 rgb(3, 169, 244)
	);
    border-radius: 10px;
    padding: 20px;
}

QWidget#content {
    background-color: #ffffff;
    border-radius: 10px;
    padding: 20px;
}

QMessageBox {	
	background-color: qlineargradient(
    	spread:pad, 
    	x1:0, y1:0, x2:1, y2:0,
   		stop:0 rgb(3, 169, 244),
    	stop:0.4 rgb(3, 169, 244),
    	stop:0.401 rgb(0, 49, 71),
   		stop:1 rgb(0, 49, 71)
	);
    border-radius: 10px;
    padding: 20px;
}

QLabel {
    background-color: #fff;
    font-size: 14px;
    color: #555;
}

QLabel#title {
    font-size: 24px;
    font-weight: bold;
    color: #333;
}

QLineEdit {
    background-color: #fff;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 10px;
    font-size: 14px;
    color: #333;
}

QLineEdit:focus {
    border: 1px solid #007BFF;
    background-color: #fff;
}

QPushButton {
    background-color: #03a9f4;
    color: white;
    border: none;
    border-radius: 10px;
    padding: 10px 15px;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #ff568c;
}

QPushButton:pressed {
    background-color: #ff568c;
}

QProgressBar {
    background-color: #fff;
    border-radius: 5px;
    text-align: center;
}

QProgressBar::chunk {
    background-color: #03a9f4;
    border-radius: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QWidget" name="content" native="true">
      <property name="minimumSize">
       <size>
        <width>525</width>
        <height>300</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>66</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="horizontalSpacing">
          <number>15</number>
         </property>
         <property name="verticalSpacing">
          <number>10</number>
         </property>
         <item row="3" column="2">
          <widget class="QPushButton" name="browse_button">
           <property name="text">
            <string>Browes</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="url_label">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="text">
            <string>URL:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="location_line_edit">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="location_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Location:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="title">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>Welcome</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="url_line_edit">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QProgressBar" name="progress_bar">
           <property name="value">
            <number>24</number>
           </property>
           <property name="format">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="download_button">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Download</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>67</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>111</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>110</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>201</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
